src/components/Header/index.tsx

```diff
@@ -59,107 +59,114 @@ export function useHeaderHeight(): number {
 
 const MenuPopover: FC = () => {
   const [t] = useTranslation('common', { keyPrefix: 'navigation' })
+  const [openMenuPanel, setOpenMenuPanel] = useState<boolean>(false)
+
   return (
-    <Popover className={styles.menuPopover}>
-      {({ close }) => (
-        <>
-          <Popover.Button as={'div'} className={clsx(styles.trigger, DISABLE_CGOL_MOUSE_CONTROLLER)}>
-            <MenuIcon />
-            <span className={styles.text}>MENU</span>
-          </Popover.Button>
+    <div className={styles.menuPopover}>
+          <>
+            <div onClick={() => setOpenMenuPanel(!openMenuPanel)} className={clsx(styles.trigger, DISABLE_CGOL_MOUSE_CONTROLLER)}>
+              <MenuIcon />
+              <span className={styles.text}>MENU</span>
+            </div>
+            {openMenuPanel && <div className={styles.menuPopoverOverlay} onClick={() => setOpenMenuPanel(false)}/>}
 
-          {/* Use Portal to prevent being influenced by mix-blend-mode */}
-          <Portal>
-            <Transition
-              as={Fragment}
-              enter={styles.enter}
-              enterFrom={styles.enterFrom}
-              enterTo={styles.enterTo}
-              leave={styles.leave}
-              leaveFrom={styles.leaveFrom}
-              leaveTo={styles.leaveTo}
-            >
-              <Popover.Panel className={styles.menuPopoverContent}>
-                <div className={styles.menu}>
-                  <StyledLink href="/developers" className={styles.title}>
-                    <CodeIcon />
-                    {t('developers')}
-                  </StyledLink>
-                  <div className={styles.links}>
-                    <StyledLink href="https://docs.nervos.org/" className={styles.link}>
-                      {t('docs')}
-                    </StyledLink>
-                    <StyledLink href="https://github.com/nervosnetwork/" className={styles.link}>
-                      {t('github')}
-                    </StyledLink>
-                    <StyledLink href="https://explorer.nervos.org/" className={styles.link}>
-                      {t('explorer')}
+            {/* Use Portal to prevent being influenced by mix-blend-mode */}
+            <Portal>
+              <Transition
+                show={openMenuPanel}
+                as={Fragment}
+                enter={styles.enter}
+                enterFrom={styles.enterFrom}
+                enterTo={styles.enterTo}
+                leave={styles.leave}
+                leaveFrom={styles.leaveFrom}
+                leaveTo={styles.leaveTo}
+              >
+            {openMenuPanel ? 
```

Comment:
This condition can be removed, because Transition's `show` prop manages its display state. Having this condition here would cause the fade effect to not work properly.